packages/vue-language-core/src/languageModule.ts

```diff
@@ -69,6 +98,29 @@ export function createLanguage(
 					}
 					return host.getScriptSnapshot(fileName);
 				},
+				resolveModuleNameLiterals(
+					moduleLiterals,
+					containingFile,
+					redirectedReference,
+					options,
+					sourceFile
+				) {
+					return moduleLiterals.map((_, index) => {
+						let moduleName = moduleLiterals[index].text;
```

Comment:
Why not use first argument?

Reply:
No reason that it can't be now. These changes started out from me looking at https://github.com/sveltejs/language-tools/blob/3f247aed28488f479399da2d852b3d258875bfbc/packages/typescript-plugin/src/module-loader.ts#L231.